### ISSUE TITLE
[Feature Request] Add option to hide icon unconditionally in the menu

### ISSUE THREAD

**Issue by Zranz** [foudfou/FireTray#170]
This feature is useful for Unity users, who already have an integrated Thunderbird icon in the system, as well as indication of new messages and notifications. A second icon is an excess, and I would like to deactivate the use of FF icon.

![2b4fd0da-b5b1-11e4-9b88-5abcaa6fcbd2](https://cloud.githubusercontent.com/assets/10563919/6572034/0df0e1b8-c70f-11e4-8896-f1f5c2fd7ba1.png)

This issue has been previously mentioned in #137, #164 and #150.

**Comment by Mrfuyu** [comment]
There's no need of this feature in my opinion. Just define under "Symbol" a user defined icon and leave this entry empty -> No icon in addition to the indicator ;)

**Comment by zranz** [comment]
Sorry, your proposed solution does not work. After leaving custom icon entry empty and closing thunderbird (actually sending the program to the tray), the default icon reappears automatically.

**Comment by Mrfuyu** [comment]
@Zranz hm sorry, it works for me :/
I use FireTray v0.6.1 with v45.2.0 Thunderbird on 16.04 Xubuntu

**Comment by zranz** [comment]
Yeah, I cannot make it disappear. I use Ubuntu 16.04, so the difference may be I am using Unity desktop. Thanks.

**Comment by Mrfuyu** [comment]
I just tested it with an 1x1px transparent png and it also worked on Xubuntu. Maybe you could try this ;). I attached the used image. Download: [https://cloud.githubusercontent.com/assets/14253537/17790634/a80d538a-6597-11e6-8cc8-57b522958d8a.png]

![1px_transparent](https://cloud.githubusercontent.com/assets/14253537/17790634/a80d538a-6597-11e6-8cc8-57b522958d8a.png)

**Comment by zranz** [comment]
Strange, it does not work. I tried different png's and none worked, default icon was present all the time.

**Comment by Mrfuyu** [comment]
Ok, that's really strange :/
What you also could try is to change the default icon under ~/.thunderbird/xxxxxx.default/extensions/firetray-id/chrome/skin/icons/linux/hicolor/22x22/apps/thunderbird.png (not sure but i think this is the default icon that is used by FireTray).
